content/100-getting-started/01-quickstart-sqlite.mdx

```diff
@@ -18,7 +18,7 @@ If you want to use Prisma ORM with your own PostgreSQL, MySQL, MongoDB or any ot
 
 ## Prerequisites
 
-You need Node.js v16.13.0 or higher for this guide (learn more about [system requirements](/orm/reference/system-requirements)).
+You need Node.js 18.18.0 / 20.9.0 / 22.11.0 or higher for this guide (learn more about [system requirements](/orm/reference/system-requirements)).
```

Comment:
Isn't mention the minimum required version enough 👁️ ?
```suggestion
You need Node.js 18.18.0 or higher for this guide (learn more about [system requirements](/orm/reference/system-requirements)).
```

Same suggestion applies for the one above.

Reply:
Afaiu the answer is actually no because e.g. a version like 19.0 or 20.1. wouldn't be supported, see [here](https://docs-v6.docs-51g.pages.dev/orm/more/upgrade-guides/upgrading-versions/upgrading-to-prisma-6#minimum-supported-nodejs-versions):

<img width="536" alt="image" src="https://github.com/user-attachments/assets/494fdfb4-a6e8-4626-aa91-8cd7acc104e6">

But I think you have a point that this may be confusing, I'll think about how to make this clearer 👍

Reply:
I opted for this now, keep it simple and link to the system requirements for specifics.

<img width="537" alt="image" src="https://github.com/user-attachments/assets/e4f9fc42-5c41-4d07-90e9-d6ba68e1a076">